examples/aws.yaml

```
@@ -280,6 +280,12 @@ sqsexporter:
     iam.amazonaws.com/role: arn:aws:iam::012345678901:role/SQSExporterRole
   imagePullSecrets:
   - name: pullsecretname
+  serviceAccount:
+    create: true
+    annotations:
+      eks.amazonaws.com/role-arn: arn:aws:iam::1234567890:role/sqs-exporter-role
```

Just for the sake of consistency, shouldn't the role ARN match the one in line 280?

```suggestion
      eks.amazonaws.com/role-arn: arn:aws:iam::012345678901:role/SQSExporterRole
```

---

Yes, sorry I missed your comment. I could address later on in another release.